src/components/slider/slider.tsx

```diff
@@ -153,9 +166,11 @@ export class Slider implements LabelableComponent, FormComponent, InteractiveCom
   componentWillLoad(): void {
     this.isRange = !!(this.maxValue || this.maxValue === 0);
     this.tickValues = this.generateTickValues();
-    this.value = this.clamp(this.value);
+    if (!Array.isArray(this.value)) {
+      this.value = this.clamp(this.value);
+    }
     afterConnectDefaultValueSet(this, this.value);
-    if (this.snap) {
+    if (this.snap && !Array.isArray(this.value)) {
       this.value = this.getClosestStep(this.value);
     }
```

Comment:
I tried doing this for range values but it was messing up some tests.

Reply:
> Can you share the test failures you noticed? It could be that we need to update the tests.

I was going to go down that route but stopped because its really out of scope. Maybe another issue if we need to clamp or set closestStep.

The failures I saw was that it was that when handles were overlapping it was automatically moving them. 5 => 10.